app/src/main/java/at/roteskreuz/stopcorona/model/db/DefaultDatabase.kt

```diff
@@ -174,21 +175,24 @@ abstract class DefaultDatabase : RoomDatabase() {
                 execSQL("ALTER TABLE `configuration_questionnaire_answer_temp` RENAME TO `configuration_questionnaire_answer`")
             },
             /**
-             * Removing [DbNearbyRecord], [DbAutomaticDiscoveryEvent].
+             * Removing [DbNearbyRecord], [DbAutomaticDiscoveryEvent], [DbSentInfectionMessage].
```

Comment:
Please use new migration for it. This one is already finished.

Reply:
updated